web/themes/custom/fmcg_material/sass/_button.scss

```diff
@@ -30,3 +30,66 @@ input.button, .btn, a.btn, span.btn, .tabledrag-toggle-weight, input[type="submi
     background: $fmcg-black;
   }
 }
+
+.button-action {
+  background-color: #50C878;
+  color: #FFFFFF;
+
+  &:hover {
+    background-color: #FFFFFF;
+    color: #50C878;
+    border-color: #50C878;
+    transition: none;
+  }
+}
+
+.ui-widget-content a, .change-role a {
+  padding: 8px 12px;
+  height: auto;
+
+  &:hover {
+    padding: 6px 10px;
+    border: 2px solid $fmcg-yellow;
+    color: $fmcg-yellow;
+    background: $fmcg-black;
+  }
+}
+
+.js .dropbutton-widget .dropbutton {
+  overflow: hidden;
+  margin: -12px;
+  padding: 0;
+  list-style-type: none;
+  list-style-image: none;
+}
+
+li.approve, li.reject {
```

Comment:
I'm just concerned about how this will work if we add a third button to the dropbutton (or even just rename one to a different name). It looks like the styles won't apply beyond the approve and reject button.

replacing this with 
```css
.dropbutton li {
```